test/ConsolueModuleLoader.test.cpp

```diff
@@ -10,6 +10,7 @@ TEST(ConsoleModuleLoader, NoModuleLoaded)
 	// Returns empty string if no module available
 	EXPECT_EQ("", consoleModLoader.GetModuleName());
 
+	EXPECT_FALSE(consoleModLoader.IsModuleLoaded());
 	EXPECT_TRUE(consoleModLoader.IsModuleLoaded(""));
```

Comment:
Should this return false instead? It is a corner case where no module is loaded (stored as an empty string). We could have an empty string always return as false. This makes sense to me because you couldn't contain the console module directories in a the directory "", as that would likely be the root directory?

Reply:
I had wondered about this too. I don't have a clear answer. I suppose you could dump a module in the root game folder. Might even be useful for single file modules. It's either a single file or a single directory, so kind of the same either way. But then, there's an assumption about the name of the module DLL within the module folder, so that breaks things.

Either way you go, you'll need to document this corner case. Or maybe consider an alternate design that avoids this corner case.

Reply:
OK, agree on returning false. I was leaning that way as well.

Thanks,
Brett